compiler/src/iree/compiler/InputConversion/MHLO/ConvertCollectiveOps.cpp

```diff
@@ -335,6 +335,162 @@ struct AllReduceOpConversion : public OpConversionPattern<mhlo::AllReduceOp> {
   }
 };
 
+Value EmitTranspose(ConversionPatternRewriter &rewriter, const Location &loc,
```

Comment:
style nit: Locations are usually passed by value as with Value
```suggestion
Value emitTranspose(ConversionPatternRewriter &rewriter, Location loc,
```